contrib/raspberrypi3/run_electrumx.sh

```diff
@@ -0,0 +1,38 @@
+###############
+# run_electrumx
```

Comment:
No, it's fine for me if you think it's useful. I was just confused because it looked like an init-replacement.